src/components/Modal/index.js

```diff
@@ -197,10 +198,14 @@ const ModalView = ({
   >
     {!!title ? <ModalHeader {...{ theme, title, unsetModal, ...props }} /> : null}
     <div
-      css={`
+      className={`${css`
         z-index: 1000;
         flex: 1;
+        display: flex;
+        flex-direction: column;
```

Comment:
this is a safe global change for the other modals?

Reply:
I went through and checked the other modals which looked fine but you're right, probably safer to have this set specifically for this